ivy/functional/backends/paddle/experimental/statistical.py

```diff
@@ -288,3 +288,18 @@ def unravel_index(
         indices = paddle.floor(indices / dim)
 
     return tuple(reversed(coord))
+
+
+def cov(
+    x1: paddle.Tensor,
+    x2: paddle.Tensor = None,
+    /,
+    *,
+    rowVar: bool = True,
+    bias: bool = False,
+    ddof: Optional[int] = None,
+    fweights: Optional[paddle.Tensor] = None,
+    aweights: Optional[paddle.Tensor] = None,
+    dtype: Optional[paddle.dtype] = None,
+) -> paddle.Tensor:
+    raise IvyNotImplementedException()
```

Comment:
Hi @abdulasiraj ,

You have to implement this rather than raising an exception here, there is another PR of your which implements the `numpy` frontend for this exact funcion.
You will first hae to complete the paddle implementation on the backend for both of these to be merge :slightly_smiling_face: